wai-extra/Network/Wai/Middleware/RequestLogger.hs

```diff
@@ -308,10 +305,7 @@ detailedMiddleware' cb ansiColor ansiMethod ansiStatusCode app req sendResponse
     collectPostParams (postParams, files) = postParams ++
       map (\(k,v) -> (k, "FILE: " <> fileName v)) files
 
-    readInt bs =
```

Comment:
This is actually different than S8.readInteger. I was assuming that was an accident, but maybe there is a purpose?